src/sentry/deletions/base.py

```diff
@@ -58,6 +60,12 @@ def get_child_relations_bulk(self, instance_list):
             # ModelRelation(Model, {'parent_id__in': [i.id for id in instance_list]})
         ]
 
+    def filter_relations(self, child_relations):
+        if not self.skip_models or not child_relations:
+            return child_relations
+
+        return filter(lambda rel: rel.params['model'] not in self.skip_models, child_relations)
```

Comment:
This isn't going to be compatible with python3 since this will be an iterable. Then when you do the truthy check, it will always be true in python3.